pkg/jobs/adopt.go

```diff
@@ -250,31 +251,83 @@ func (r *Registry) resumeJob(
 ) (retErr error) {
 	log.Infof(ctx, "job %d: resuming execution", jobID)
 
-	args := []interface{}{jobID, s.ID().UnsafeBytes(),
-		r.clock.Now().GoTime(), r.RetryInitialDelay(), r.RetryMaxDelay()}
+	job, err := r.loadJobForResume(ctx, jobID, s)
+	if err != nil {
+		return err
+	}
+
+	// A nil job return means we loaded the job but it isn't time
```

Comment:
nit: i'm not sure what this means. Maybe in the loadJobForResume docstring, you could describe the situations that result in a nil job without an err?